rest/model/patch.go

```diff
@@ -172,6 +173,13 @@ func (apiPatch *APIPatch) BuildFromService(h interface{}) error {
 
 	apiPatch.PatchedConfig = utility.ToStringPtr(v.PatchedConfig)
 	apiPatch.CanEnqueueToCommitQueue = v.HasValidGitInfo()
+
+	childPatches := make([]*string, 0)
```

Comment:
There's a function utility.ToStringPtrSlice you can use for this

Reply:
Thanks! That's much neater. done.